subiquity/common/types.py

```diff
@@ -391,7 +391,7 @@ class TimeZoneInfo:
 
 @attr.s(auto_attribs=True)
 class UbuntuAdvantageInfo:
-    token: str
+    token: str = attr.ib(default='', repr=False)
```

Comment:
can we avoid the default value?

Reply:
Yes, I wasn't trying to change behavior on that point.